### PR TITLE
Added Cypress Spec for asset creation

### PR DESCRIPTION
**Pull request by GokulramGHV**
## Proposed Changes
- Added a cypress spec for asset creation and update

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by GokulramGHV** [comment]
Part of #4133

**Comment by nihal467** [comment]
@GokulramGHV can you check the merge conflict and once its done, mark it for testing

**Comment by nihal467** [comment]
@GokulramGHV check the merge conflict

**Comment by nihal467** [comment]
@GokulramGHV check the merge conflict and mark it for testing

File: src/Components/Facility/AssetCreate.tsx [hunk 1]
```diff
@@ -242,7 +242,9 @@ const AssetCreate = (props: AssetProps) => {
             errors[field] = "Field is required";
             invalidForm = true;
           }
+          console.log("Phone String: ", support_phone);
```
Comment (khavinshankar) [inline src/Components/Facility/AssetCreate.tsx]:
```suggestion
```

**Review by khavinshankar — APPROVED** [review]

**Comment by khavinshankar** [comment]
@GokulramGHV can you checkout the cypress fails, likely merging the develop branch should fix it

**Comment by nihal467** [comment]
@GokulramGHV check the cypress failed

**Comment by GokulramGHV** [comment]
@khavinshankar Can you please check why the cypress is failing?

**Comment by nihal467** [comment]
@GokulramGHV work on it, and once it is fully ready, mark it for need testing

**Comment by nihal467** [comment]
@GokulramGHV what is the status of the PR

**Comment by GokulramGHV** [comment]
@nihal467 I'll complete it soon.. I was waiting for the DateInput Field bug to be fixed

**Comment by nihal467** [comment]
@GokulramGHV what is the status on this PR

**Comment by nihal467** [comment]
@GokulramGHV what is the status on this PR

**Comment by khavinshankar** [comment]
@GokulramGHV can you clear the merge conflicts, I'll review this and merge it

**Comment by ShivamJhaa** [comment]
@khavinshankar @nihal467 The PR seems to be stale, can I take up the task?

**Pull request closed without merging**